nss-day-cohort-29/reactive-nutshell-starrywick
# Setup zc

This is not ready to be tested as of yet. I am going to leave this open and continue committing. I will update when available to fetch.

*** UPDATE

Description:
TaskList and deleteTask are working

Type of change

[X ] New feature (non-breaking change which adds functionality)

Testing Instructions for Change Made:
1. git fetch --all
1. git checkout branch setup_zc
1. run json server on port 5002
1. npm start in reactive-nutshell
1. Add the data below to database json;
```
"tasks": [
    {
      "id": 1,
      "userId": 2,
      "task": "task",
      "dueDate": "2018-01-28",
      "complete": false
    },
    {
      "id": 2,
      "userId": 2,
      "task": "task2",
      "dueDate": "2019-01-28",
      "complete": true
    }
  ]
```
1. A list of two tasks should appear.
1. click on tasks in navbar
1. try deleting a task

Checklist:
[X ] My code follows the style guidelines of this project
[X ] I have performed a self-review of my own code
[X ] My changes generate no new warnings
[X ] I have added test instructions that prove my fix is effective or that my feature works
